src/app/src/components/FacilityListItemsConfirmationTableRow.jsx

```diff
@@ -110,7 +110,9 @@ function FacilityListItemsConfirmationTableRow({
                     variant="head"
                     style={listTableCellStyles.headerCellStyles}
                 >
-                    <b>Actions</b>
+                    <ShowOnly when={!readOnly}>
+                        <b>Actions</b>
+                    </ShowOnly>
```

Comment:
This is a change from how we displayed the read-only page before for admins, but it seemed a little weird to me to have an empty "Actions" header when we hide the actions themselves